<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>logpassbase</class>
  <widget class="QDialog" name="logpassbase" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>334</width>
        <height>157</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Usuario</string>
    </property>
    <widget class="QLabel" name="textLabel2" >
      <property name="geometry" >
        <rect>
          <x>10</x>
          <y>70</y>
          <width>90</width>
          <height>16</height>
        </rect>
      </property>
      <property name="text" >
        <string>Contraseña</string>
      </property>
    </widget>
    <widget class="QLabel" name="textLabel1_2" >
      <property name="geometry" >
        <rect>
          <x>10</x>
          <y>100</y>
          <width>120</width>
          <height>26</height>
        </rect>
      </property>
      <property name="font" >
        <font>
          <family>Flubber</family>
          <pointsize>20</pointsize>
          <weight>0</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
        </font>
      </property>
      <property name="text" >
        <string>BulmaGés</string>
      </property>
      <property name="textFormat" >
        <enum>Qt::PlainText</enum>
      </property>
    </widget>
    <widget class="QLabel" name="textLabel1" >
      <property name="geometry" >
        <rect>
          <x>10</x>
          <y>40</y>
          <width>120</width>
          <height>16</height>
        </rect>
      </property>
      <property name="text" >
        <string>Nombre Usuario</string>
      </property>
    </widget>
    <widget class="QLineEdit" name="m_login" >
      <property name="geometry" >
        <rect>
          <x>160</x>
          <y>40</y>
          <width>160</width>
          <height>20</height>
        </rect>
      </property>
    </widget>
    <widget class="QLineEdit" name="m_password" >
      <property name="geometry" >
        <rect>
          <x>160</x>
          <y>70</y>
          <width>161</width>
          <height>20</height>
        </rect>
      </property>
      <property name="echoMode" >
        <enum>QLineEdit::Password</enum>
      </property>
    </widget>
    <widget class="QLabel" name="textLabel1_2_2" >
      <property name="geometry" >
        <rect>
          <x>16</x>
          <y>124</y>
          <width>100</width>
          <height>22</height>
        </rect>
      </property>
      <property name="font" >
        <font>
          <family>Flubber</family>
          <pointsize>0</pointsize>
          <weight>0</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
        </font>
      </property>
      <property name="text" >
        <string>Lanzador v.0.5</string>
      </property>
      <property name="textFormat" >
        <enum>Qt::PlainText</enum>
      </property>
    </widget>
    <widget class="QPushButton" name="pushButton24" >
      <property name="geometry" >
        <rect>
          <x>160</x>
          <y>100</y>
          <width>160</width>
          <height>35</height>
        </rect>
      </property>
      <property name="text" >
        <string>Validar</string>
      </property>
      <property name="autoDefault" >
        <bool>false</bool>
      </property>
      <property name="default" >
        <bool>false</bool>
      </property>
    </widget>
    <widget class="QLabel" name="lblAuthError" >
      <property name="enabled" >
        <bool>true</bool>
      </property>
      <property name="geometry" >
        <rect>
          <x>10</x>
          <y>10</y>
          <width>312</width>
          <height>21</height>
        </rect>
      </property>
      <property name="text" >
        <string/>
      </property>
      <property name="textFormat" >
        <enum>Qt::PlainText</enum>
      </property>
      <property name="alignment" >
        <set>Qt::AlignCenter</set>
      </property>
    </widget>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <tabstops>
    <tabstop>m_login</tabstop>
    <tabstop>m_password</tabstop>
    <tabstop>pushButton24</tabstop>
  </tabstops>
</ui>
